util/opentelemetry-util-http/src/opentelemetry/util/http/__init__.py

```diff
@@ -228,20 +235,44 @@ def get_custom_headers(env_var: str) -> list[str]:
     return []
 
 
-def _parse_active_request_count_attrs(req_attrs):
-    active_requests_count_attrs = {
-        key: req_attrs[key]
-        for key in _active_requests_count_attrs.intersection(req_attrs.keys())
+def _parse_active_request_count_attrs(
```

Comment:
`opentelemetry-util-http` is a utility package used only for http helper functions. We do not want to mix semantic convention logic in here. I suggest creating a method in the instrumentation file itself and calling the helper method within `opentelemetry-instrumentation` like we did in [asgi](https://github.com/open-telemetry/opentelemetry-python-contrib/blob/main/instrumentation/opentelemetry-instrumentation-asgi/src/opentelemetry/instrumentation/asgi/__init__.py#L934).